README.md

```diff
@@ -1,14 +1,12 @@
 # kyt-cli
```

Comment:
Move sentence "Get the iot hub connection string from the Azure Portal. Select the iot hub, then "shared access policies"" into server docker section,

Reply:
Moved and rephrased: 
Get the IoT Hub connection string from the Azure Portal. Select the IoT Hub, then "shared access policies". Copy from `iothubowner` the connection string `Connection string—primary key`.